### PR TITLE
Laravel Upgrade

### PR DESCRIPTION
**Pull request by rajat-agarwal1**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,12 +15,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "~4.0|~5.0|~6.0|~7.0|~8.0|~9.0",
```
Comment (rajeevrai) [inline composer.json:18]: do not remove 9.0.. just add 10.0 as well

File: composer.json [hunk 2]
```diff
@@ -15,12 +15,12 @@
...
+        "illuminate/support": "~4.0|~5.0|~6.0|~7.0|~8.0|~10.0",
         "fakerphp/faker": "~1.4"
     },
     "require-dev": {
         "phpspec/phpspec": "~2.0",
-        "illuminate/database": "~4.0|~5.0|~6.0|~7.0|~8.0|~9.0",
```
Comment (rajeevrai) [inline composer.json:23]: same here

**Review by rajeevrai — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (rajat-agarwal1) [reply composer.json:18]: done

File: composer.json (hunk 2 above)
Comment (rajat-agarwal1) [reply composer.json:23]: done

**Review by rajeevrai — APPROVED** [review]

**Pull request merged**